### PR TITLE
Add Embit devices

### PR DESCRIPTION
**Pull request by andguaEmbit**
#### Summary

Add Embit devices

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

**Comment by benolayinka** [comment]
Thanks for submitting your devices @andguaEmbit ! It looks like your files are failing formatting. Please run `make validate fmt` locally, and 'make validate' to check against the schema, as per the [`README`](https://github.com/TheThingsNetwork/lorawan-devices/blob/master/README.md)

Also, .yaml and .js files must have lowercase titles. @johanstokking , we need to add this to the documentation.

File: vendor/Embit/EMB-LR1280-mPCI-4x.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: EMB-LR1280-mPCI-4x
```
Comment (johanstokking) [inline vendor/Embit/EMB-LR1280-mPCI-4x.yaml:1]: This needs to be lowercase

File: vendor/Embit/EMB-LR1280-mPCI-4x.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+description: EMB-LR1280-mPCIe-4x provides long range connectivity and high interference immunity exploiting spread spectrum modulation over the 2.4 GHz ISM band. It can operate worldwide being fully compliant with 2.4 GHz frequency band regulation.
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '1.0'
+    numeric: 1
```
Comment (johanstokking) [inline vendor/Embit/EMB-LR1280-mPCI-4x.yaml:8]: We need a profile reference here

File: vendor/Embit/EMB-LRWL55.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '1.0.1'
+    numeric: 1
+
+# LoRaWAN Device Profiles per region
+  # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867, RU864-870
+profiles:
```
Comment (johanstokking) [inline vendor/Embit/EMB-LRWL55.yaml:12]: This falls under the firmware version, so it needs to be properly indented

File: vendor/Embit/EMB-LRWL55.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  EU863-870:
+    # Unique identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+    id: EMB-LRWL55-profile-eu868
```
Comment (johanstokking) [inline vendor/Embit/EMB-LRWL55.yaml:15]: Also these references need to be lowercase

**Review by johanstokking — CHANGES_REQUESTED** [review]

**Pull request merged**